src/components/avatar/avatar.jsx

```diff
@@ -0,0 +1,38 @@
+import { ThemeProvider } from "styled-components";
+import styled from "styled-components";
+
+/**
+ * Avatar Component
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param shape expects 'round' or 'square'
+ * @param src the image link
+ * @param size expects 'xs', 'small', 'medium', or 'large'
+ * @param alt the image description
+ */
+
+const Container = styled.img`
+  height: ${(props) => props.theme[props.size]};
+  width: ${(props) => props.theme[props.size]};
+`;
+
+export const Avatar = ({ shape, src, size, alt }) => {
+  const sizes = {
+    xs: "35px",
+    small: "40px",
```

Comment:
Nit pick-- since we're using bootstrap, there are many other things in bootstrap that utilize sizes like this, however they use 2-letter codes for sizes. It would be more consistent if you use
```
{
xs: ...
sm: ...
md: ...
lg: ...
}
```